test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -37,6 +37,15 @@ git clone -b ${SMDEBUG_RELEASE_TAG} https://github.com/awslabs/sagemaker-debugge
 echo "Changing to sagemaker-debugger."
 cd sagemaker-debugger
 
+# Temporary Change:
+# Run Tests on a custom branch for forwards compatibility
+# For TF 2.2.0. Revert after DLC release
+TF_VERSION=$(python -c "import tensorflow as tf; print(tf.__version__)")
+if [[ $TF_VERSION == "2.2.0" ]]; then
+    git checkout save_inputs_asserts # temporary branch change
```

Comment:
nit: will we also need this for 2.3?

Reply:
We do not need this for 2.3

```
echo "Running ZCC tests."
if [[ $TF_VERSION != "2.3.0" ]]; then
    timeout 2100s pytest mock/tests/zero_code_change/${ZCC_TEST_NAME[$FRAMEWORK]}
fi
```

This snippet here should handle 2.3 for now